servers/visual/visual_server_viewport.cpp

```diff
@@ -128,7 +128,7 @@ void VisualServerViewport::_draw_viewport(Viewport *p_viewport, ARVRInterface::E
 
 			for (Set<RasterizerCanvas::Light *>::Element *F = canvas->lights.front(); F; F = F->next()) {
 				RasterizerCanvas::Light *cl = F->get();
-				if (cl->enabled && cl->texture.is_valid()) {
+				if (cl->enabled && cl->texture.is_valid() && cl->energy > 0.0f) {
```

Comment:
Should this maybe be using something like `is_zero_approx()` instead to deal with floating point errors? Not sure.

Reply:
True. Using exactly 0 will fail to trigger it due to floating point errors

Reply:
It triggers even on ARM platform (I checked for perfomance gain in project I'm working on), but I agree that it will be safer to compare with epsilon.